### ISSUE TITLE
Widnows updates dosent work anymore

### ISSUE THREAD

**Issue by SoXX-TheFennec** [Disassembler0/Win10-Initial-Setup-Script#86]
After I run the Script WIndows is not able to update anymore. Every time I open the Update menu windows is telling me that my Device is in danger and it needed to be updated. Has this something to do with the Script?

I used the default settings.

**Comment by Disassembler0** [comment]
https://github.com/Disassembler0/Win10-Initial-Setup-Script/blob/f3b74a650434d1421db2465c500d0d76b8968bc2/Win10.ps1#L164-L167

The FAQ explicitly says not to run the script if you don't know what does it do. There are other, much nicer tools (even with GUI) for less tech savvy people. This is not one of them.

**Comment by SoXX-TheFennec** [comment]
Oh sorry, overread this then. I know some thing the Script dose to WIdnows but I failed to realize that. But thanks for the very quick response :D

**Comment by SoXX-TheFennec** [comment]
@Disassembler0 I completely read your script comments and FAQ and tested it again with Windows 10 Home edition. The same issue occurs there. Could it be that this issue is also present in newer versions of Win10 Home? I can provide you the Windows Version number once I am back home in case you need it.

**Comment by Disassembler0** [comment]
Sure, it could be. It could be even that the issue is present in all editions and happens if you try to install sufficiently old build and apply the tweak immediately after installation, before it even manages to connect online and update WU components for the first time.

The remark about Enterprise is there due to issue #57 as the problem was first observed on Enterprise which were in service for some time already and suddenly stopped receiving updates with the same message as you're getting.

**Comment by T4CC** [comment]
If I am not mistaken, this goes back to 2015/16, where disabling telemetry causes Windows Updates to go off the rails.  I browse the various privacy script for updating my own and generally avoid any posting, but the least I can do is share a possible fix.

In my own privacy script, Windows Update fails after disabling telemetry unless I add:
Path : hklm:\SOFTWARE\Policies\Microsoft\Windows\PreviewBuilds
ValueName : AllowBuildPreview 
ValueType : DWord
Value : 0

Hopefully that helps.

TLDR:
When you preview builds and become guinea pigs, your doing so at own risk and agreeing to give away your firstborn child.

Also, on page 17,348,213, subsection 9090, paragraph NOP of the agreement, you allowed them have all your data, 'ANONYMIZED' (lol) of coarse, that they deem they want/require/can monetize/etc.

Because you agreed to give the last shards of your privacy away for Beta software, when you disable telemetry and are still a guinea pig/beta tester/previewer, there is no soup for you. 

By disabling preview builds, you gain your right to the illusion of privacy back, and updates are allowed again.
